data/pirate jobs.txt

```diff
@@ -0,0 +1,2750 @@
+		ship "Quicksilver (Scanner)"
+	npc
+		system destination
+		distance 1
+		personality heroic waiting surveillance
+		government Syndicate
+		ship "Quicksilver (Scanner)"
+	npc
+		system destination
+		personality heroic staying surveillance
+		government Syndicate
+		ship "Quicksilver (Scanner)"
+		ship "Quicksilver (Scanner)"
+	on complete
+		payment 500000 25000
+		dialog "You land outside of the main city to avoid any chances of your passenger being seen. They grab their things and make their way off of your ship after paying you <payment>."
+
+mission "Highly Wanted Passenger (South)"
+	name "Galactic criminal to <planet>"
+	job
+	repeat
+	infiltrating
+	description "This criminal is wanted by every law enforcement agency in the galaxy and needs safe transport to <destination>. The payment is high at an amazing <payment>, but transporting such a person could lead to trouble."
+	passengers 1
+	cargo "Illegal Cargo" 1
+		illegal 1000000 `Law enforcement rush their way onto your ship through the airlocks, suited head to toe in armor and armed to the teeth. Your crew is quickly made to surrender and your passenger is hauled off in cuffs. The officers consider detaining you and your crew as well, but eventually decide to only fine you to reduce the trouble they have to go through.`
+		stealth
+	to offer
+		random < 15
+		or
+			"reputation: Pirate" > 10
+			"combat rating" > 1000
+	source
+		government "Pirate"
+		attributes "south pirate"
+	destination
+		distance 4 14
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Pirate"
+	npc
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		distance 5
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		distance 4
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		distance 3
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		system destination
+		distance 1
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		system destination
+		distance 1
+		personality heroic waiting surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+	npc
+		system destination
+		personality heroic staying surveillance
+		government Militia
+		ship "Bastion (Scanner)"
+		ship "Bastion (Scanner)"
+	on complete
+		payment 500000 25000
+		dialog "You land outside of the main city to avoid any chances of your passenger being seen. They grab their things and make their way off of your ship after paying you <payment>."
+
+mission "Slave Transport [0]"
+	name "Move slaves to <planet>"
+	job
+	repeat
+	description "These pirates need <bunks> slaves moved to <destination> along with <cargo>. Payment is <payment>. Slavery is highly illegal throughout the galaxy, so this may be a dangerous job to take."
+	passengers 5 5 .2
+	cargo "Illegal Cargo" 1 5
```

Comment:
Rather than all these illegal passengers having a generic "Illegal Cargo" to make it so you have to avoid scans, it might make more sense to just name the cargo something that goes with the mission, like "personal effects" for the criminals or "shackles" for the slaves. Also, the name of the cargo should be lower-case, just for consistency.